.github/workflows/build_linux_packages.yml

```diff
@@ -68,6 +73,8 @@ jobs:
           pip install -r requirements.txt
 
       - name: Build Projects
+        id: build
+        continue-on-error: ${{ inputs.expect_failure }}
```

Comment:
To make it more clear in the UI, how about putting `${{ inputs.expect_failure }}` in the name too?

Getting strings out of yaml values is tricky... maybe one of these:

* `Build Projects (expect failure)`
* `Build Projects (xfail: 1)`
* `Build Projects (xfail: 0)`
* `Build Projects (xfail: true)`
* `Build Projects (xfail: false)`